src/Events/Custom_Tables/V1/Activation.php

```diff
@@ -86,6 +86,35 @@ public static function init() {
 		}
 	}
 
+	/**
+	 * Includes the Migration state into the System Information for support.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $info
+	 *
+	 * @return array
+	 */
+	public static function filter_include_migration_to_system_info( array $info = [] ): array {
+		$phase = tribe( State::class )->get_phase();
+		$incomplete_label = esc_html__( 'Incomplete', 'the-events-calendar' );
```

Comment:
Should we add something about what state it _is_ in here? Might be useful to know they are in the middle of a reversion, etc

Reply:
Also - translations should always use _x() functions - add some context for the translators 🙏🏼

Reply:
Not for the which state, but the `_x()` is a good idea

Reply:
I decided to go with untranslated strings since this is only for us.